### DISCUSSION TITLE
Includle Logi Options+ in homebrew-cask-drivers

### DISCUSSION THREAD

**Discussion by caizixian** [Homebrew/discussions#3565] [Casks] [answered]
I saw this discussion. https://github.com/Homebrew/discussions/discussions/2492

I've just checked and Logi Options+ has now stabilized (https://support.logi.com/hc/articles/1500005516462) and is publicly accessible (https://download01.logi.com/web/ftp/pub/techsupport/optionsplus/logioptionsplus_installer.zip). Maybe we could reconsider its inclusion in homebrew-cask-drivers

**Comment by mystygage** [comment, marked as answer]
It's available: `brew info logi-options-plus`
Formula: https://github.com/Homebrew/homebrew-cask-drivers/blob/HEAD/Casks/logi-options-plus.rb
